cardigan_beta/stories/global/palette/README.md

```diff
@@ -0,0 +1 @@
+__TODO:__ add palette README info to stories/global/palette/README.md.
```

Comment:
As this is public, should we be doing this?

Reply:
Good point. Have commented out.